examples/js/loaders/MD2Loader.js

```diff
@@ -38,61 +38,61 @@ THREE.MD2Loader.prototype = {
 		var normals = [
 			[ -0.525731,  0.000000,  0.850651 ], [ -0.442863,  0.238856,  0.864188 ],
 			[ -0.295242,  0.000000,  0.955423 ], [ -0.309017,  0.500000,  0.809017 ],
-			[ -0.162460,  0.262866,  0.951056 ], [  0.000000,  0.000000,  1.000000 ],
-			[  0.000000,  0.850651,  0.525731 ], [ -0.147621,  0.716567,  0.681718 ],
-			[  0.147621,  0.716567,  0.681718 ], [  0.000000,  0.525731,  0.850651 ],
-			[  0.309017,  0.500000,  0.809017 ], [  0.525731,  0.000000,  0.850651 ],
-			[  0.295242,  0.000000,  0.955423 ], [  0.442863,  0.238856,  0.864188 ],
-			[  0.162460,  0.262866,  0.951056 ], [ -0.681718,  0.147621,  0.716567 ],
+			[ -0.162460,  0.262866,  0.951056 ], [ 0.000000,  0.000000,  1.000000 ],
```

Comment:
Hmm, not a big fan of losing these nicely formatted bits of code due to automated mass re-formatting 😕